t/01_File-Fetch.t

```diff
@@ -16,7 +17,7 @@ use_ok('File::Fetch');
 $File::Fetch::DEBUG = $File::Fetch::DEBUG   = 1 if $ARGV[0];
 $IPC::Cmd::DEBUG    = $IPC::Cmd::DEBUG      = 1 if $ARGV[0];
 
-$File::Fetch::FORCEIPV4=1;
+{ no warnings 'once'; $File::Fetch::FORCEIPV4=1; }
```

Comment:
The previous line shows another mechanism for suppressing the warning, which could also be used here for consistency.